core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/PrometheusRecorder.scala

```diff
@@ -61,7 +61,9 @@ case class PrometheusRecorder(kamon: PrometheusReporter, config: MetricConfig)
   private val promMetrics = PrometheusMetrics()
 
   override def processActivation(activation: Activation, initiator: String, metricConfig: MetricConfig): Unit = {
-    lookup(activation, initiator).record(activation, initiator, metricConfig)
+    if (!metricConfig.ignoredNamespaces.contains(activation.namespace)) {
```

Comment:
I would remove then the `if statement` from inside the [record method](https://github.com/apache/openwhisk/blob/master/core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/PrometheusRecorder.scala#L130).
And I would do the same for `KamonRecorder`.

Reply:
Thanks @selfxp. One more thing, I realized with this change `openwhisk_namespace_activations_total` metric is also not updated. I think this metric should be updated. I will try to extract this metric to some other place possibly as part of global metrics updates.